src/test/unit/BLSOperatorStateRetrieverUnit.t.sol

```diff
@@ -114,7 +101,7 @@ contract BLSOperatorStateRetrieverUnitTests is MockAVSDeployer {
         assertEq(checkSignaturesIndices.nonSignerQuorumBitmapIndices.length, 0);
         assertEq(checkSignaturesIndices.quorumApkIndices.length, allInclusiveQuorumNumbers.length);
         assertEq(checkSignaturesIndices.totalStakeIndices.length, allInclusiveQuorumNumbers.length);
-        assertEq(checkSignaturesIndices.nonSignerStakeIndices.length, 0);
+        assertEq(checkSignaturesIndices.nonSignerStakeIndices.length, allInclusiveQuorumNumbers.length);
```

Comment:
could you add some revert messages to these? also helps document exactly what it's checking (although these seem ~fairly clear)